### PR TITLE
GET queries for resources one edge away from a specified resource

### PR DESCRIPTION
**Pull request by rishi-tripathy**

### CODE REVIEW

File: Models.py
Comment (rishi-tripathy) [inline Models.py:93]: Current way of making events doesn't add this edge

File: main.py
Comment (jwei98) [inline main.py]: very nice check

File: main.py
Comment (jwei98) [inline main.py]: Up to you, but prefer if/elif/elif just to indicate it can only be one of these

File: main.py
Comment (jwei98) [inline main.py]: nice use of generic output!

File: main.py
Comment (jwei98) [inline main.py]: I don't think `person_id` can be None anymore, so we can remove this section!

File: main.py
Comment (jwei98) [inline main.py]: I don't think `circle_id` can be None anymore, so we can remove this section.

File: main.py
Comment (jwei98) [inline main.py]: I don't think `event_id` can be None anymore, so we can remove this section.

File: main.py
Comment (jwei98) [inline main.py]: Can a single event belong to more than one circle?

File: main.py
Comment (jwei98) [inline main.py]: Can you check the API on this? I think we want to return whether the person is attending or not as well, right? So maybe just a list of [{person_id : True}, {person_id: False}]. Idk about this exact format.

File: Models.py
Comment (jwei98) [reply Models.py:93]: wdym? It looks like it worked in neo4j dev db?

**Review by jwei98 — CHANGES_REQUESTED** [review]
Really good job overall! A couple things here and there but very solid overall. Thank Rish!

One stylistic note: Instead of doing string checks like `if resource not in ['circles', 'events', 'knows']` and checking `if resource == 'circles'` and stuff like that, set constants at the top of the file like `CIRCLES = 'circles'`, `EVENTS = 'events'`, etc. Then say `if resource not in [CIRCLES, EVENTS, KNOWS]` and such. Generally that's what people do if they're running a bunch of checks on strings-- think it might be to avoid any sort of spelling errors.

File: main.py
Comment (jwei98) [reply main.py]: Ooof okay. Sorry. Upon second look, I wouldn't use an `output` variable here. If you know what resource they're requesting you know it can't be any other resource. So in each clause, just `return [k.json_repr() for k in person.Knows]`, `return [p.json_repr() for p in event.InvitedTo]`, etc.

File: main.py
Comment (rishi-tripathy) [reply main.py]: Can't remove this-- the optional value is the resource being requested

absence of a resource will give the object itself (/jwei/) versus the presence of a resource will give something that is an edge away (/jwei/circles)

File: main.py
Comment (rishi-tripathy) [reply main.py]: Can't remove this-- the optional value is the resource being requested

absence of a resource will give the object itself (/jwei/) versus the presence of a resource will give something that is an edge away (/jwei/circles)

File: main.py
Comment (rishi-tripathy) [reply main.py]: Can't remove this-- the optional value is the resource being requested

absence of a resource will give the object itself (/jwei/) versus the presence of a resource will give something that is an edge away (/jwei/circles)

File: main.py
Comment (rishi-tripathy) [inline main.py]: @jwei98 please check

File: main.py
Comment (rishi-tripathy) [reply main.py]: (Y)

File: main.py
Comment (jwei98) [inline main.py]: Please check this before merging. Also add a space on both sides of `=` please!

Otherwise, rest of PR looks great! Thanks!

Comment (jwei98) [reply main.py]: Please don't merge without checking this. The rest looks great! Thanks!

**Review by jwei98 — APPROVED** [review]

**Pull request merged**
